spec/features/post_peep_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'posting peeps' do
```

Comment:
This scenario doesn't involve the user signing in but you're testing that "a registered user can post peeps".

Reply:
Oh, you're right here, Eddie. Actually this surprises me... I tried to post a peep manually without being authenticated, and it does not allow me (expected) as it depends on the User object to do it... 
My guess is that after populating the table with a User (when running the setup_test_db Rake task), it authenticates the user automatically! And that is why it allows the test to post that peep and sees the content on the page. Does it make sense? I am actually quite surprise with this case, happy to go over it together next week too! Thanks.